compiler/one-cmds/one-quantize

```diff
@@ -130,6 +130,12 @@ def _get_parser():
         help=
         "Path to the quantization configuration file."
     )
+    quantization_group.add_argument(
+        '--test_data',
+        type=str,
+        help=
+        "Path to the test data. Evaluation result will be printed if test data is given."
```

Comment:
(trivail)
```suggestion
        "Data path to test. Evaluation result will be printed if test data is given."
```

Reply:
"Path to XX" is the style we've used in other options. And I'm not sure"Data path" is a widely used term.